### ISSUE TITLE
Hello，why next_client_controls is set like this？

### ISSUE THREAD

**Issue by LREM-C** [SZU-AdvTech-2022/335-SCAFFOLD-Stochastic-Controlled-Averaging-for-Federated-Learning#1]
#next_client_controls = [copy.deepcopy(params.grad.data) for params in net.parameters()]

    net.zero_grad()
    for images, labels in DataLoader(dataset=data, batch_size=args.test_batch, shuffle=True):
        images = images.to(args.device)
        labels = labels.to(args.device)
        predicts = net(images)
        loss = args.cost(predicts, labels)
        loss.backward()

    next_client_controls = [copy.deepcopy(params.grad.data)/(len(data)/args.test_batch) for params in net.parameters()]
    return net, sum(Loss)/len(Loss), next_client_controls
